cupyx/scipy/spatial/distance.py

```diff
@@ -114,8 +114,8 @@ def __init__(self, canonical_name=None, aka=None,
 def check_soft_dependencies():
     if not cuvs_available:
         if not pylibraft_available:
-            raise RuntimeError('cuVS >= 24.12 or pylibraft < '
-                               '24.12 should be installed to use this feature')
+            raise RuntimeError('cuVS >= 25.02 or pylibraft < '
+                               '25.02 should be installed to use this feature')
```

Comment:
I think pylibraft 24.12 does not have the function, so there may not be a working option for the 24.12 release!

(There is [no `distance` folder in here](https://github.com/rapidsai/raft/tree/branch-24.12/python/pylibraft/pylibraft) in the 24.12 tag, but it is there for 24.10 and before)
                               '25.02 should be installed to use this feature')

Reply:
Good point Greg. Let's let Corey weigh in on how to handle 24.12

Reply:
right, we might have to explicitly raise an exception if only 24.12 is available

Reply:
Looks like just reverting to the original error is the right approach. So have done that